src/CoursePageResources/index.scss

```diff
@@ -0,0 +1,48 @@
+.container-fluid {
+    background-color: #EAF3FB;
+    color: #2E8ED3;
+}
+
+.container-fluid-container {
+    border-color: #B2D5EE;
+}
+
+.container-fluid-enabled-button {
+    border-color: #0075B4;
+    color: #2E8ED3;
+}
+
+.container-fluid-enabled-button:hover {
+    color: inherit;
+}
+
+.sub-container-text {
+    color: #39AAE3;
+}
+
```

Comment:
These names are very generic and tied to bootstraps naming scheme instead of being specific to this app.